src/reusecore/Search/index.js

```diff
@@ -40,7 +40,7 @@ const SearchBox = ({
           debounceTimeout={500}
           onChange={(e) => handleChange(e)}
           placeholder="Search..."
-          autoFocus
+          // autoFocus
```

Comment:
One thing to say here though @balagowda, if we comment autoFocus then one inconsistency we can observe [here](https://6554d23a21c5da1a6c9369a2--layer5.netlify.app/blog) while searching and typing for any particular blog. You can compare this with the layer5.io/blog [here](https://layer5.io/blog). Can we do something so that our refresh redirection issue as well as this typing inconsistency both get resolve @balagowda ?

Reply:
I'll look into this